test/e2e/framework/framework.go

```diff
@@ -73,6 +74,7 @@ type Framework struct {
 
 	clientConfig                     *rest.Config
 	KoordinatorClientSet             koordinatorclientset.Interface
+	NRTClientSet                     nrtclientset.Interface
```

Comment:
It was previously rejected to initialize ElasticQuota's ClientSet here, but now there is an NRT Client. It is necessary to discuss whether it is necessary to initialize these ClientSets in one place, or to write some helper functions and call them in the E2E Case as needed?

Reply:
fixed